docker/mongo/Dockerfile

```diff
@@ -9,11 +9,21 @@ RUN apt-get -y update && \
     echo "deb [arch=amd64,arm64] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.2 multiverse" | tee /etc/apt/sources.list.d/mongodb-org4.2.list && \
     apt-get -y update && \
     apt-get install -y mongodb-org && \
+    apt-get install -y python3-pip && \
+    apt-get install -y supervisor && \
     rm -rf /var/lib/apt/lists/*
     
+RUN mkdir -p /var/log/supervisor
+COPY supervisord.conf /etc/supervisor/conf./supervisord.conf
 VOLUME ["/data/db"]
-WORKDIR /data
-CMD ["mongod", "--bind_ip_all"]
+VOLUME ["/var/backups/mongodb"]
 
 EXPOSE 27017
 EXPOSE 28017
+
+WORKDIR /app
+COPY requirements.txt /app
+RUN pip3 install -r requirements.txt
+COPY . /app
```

Comment:
Wouldn't it better to run `pip3 install -r requirements.txt` first and then copy everything into `/app`?
```
 WORKDIR /app

 RUN pip3 install -r requirements.txt
 COPY . /app
```

Reply:
Its not possible this way, because theres not requirements.txt file in current folder. 
This will work however, thanks:
```
WORKDIR /app
COPY . /app
RUN pip3 install -r requirements.txt
```